frontend/src/app/patients/personSchema.ts

```diff
@@ -257,12 +255,7 @@ const updateFieldSchemata: (
       [...getValues(ETHNICITY_VALUES), "", null],
       t("patient.form.errors.ethnicity")
     ),
-  gender: yup
-    .mixed()
-    .oneOf(
-      [...getValues(GENDER_VALUES), "", null],
-      t("patient.form.errors.gender")
-    ),
+  gender: yup.mixed().required(t("Gender selection is required")),
```

Comment:
Unless we're certain that this is not a required field for self-registration, this error message should remain i18n-ized since it may be seen by a patient directly.

Should we keep enforcing that this field be one of the values in the `GENDER_VALUES` constant as well?

Reply:
Also, maybe this error needs to be `Sex assigned at birth is required` or the like? @tim-skylight @Rebecca-LM what phrasing do you think best matches the intent of this field?

Reply:
Note: shunting this to the #3163 ticket for now